src/Tribe/RSVP.php

```diff
@@ -2037,44 +2074,50 @@ public function get_attendee( $attendee, $post_id = 0 ) {
 			}
 		}
 
-		$attendee_data = array_merge( $this->get_order_data( $attendee->ID ), array(
-			'optout'             => $optout,
-			'ticket'             => $product_title,
-			'attendee_id'        => $attendee->ID,
-			'security'           => $security,
-			'product_id'         => $product_id,
-			'check_in'           => $checkin,
-			'order_status'       => $status,
-			'order_status_label' => $status_label,
-			'user_id'            => $user_id,
-			'ticket_sent'        => $ticket_sent,
-
-			// Fields for Email Tickets
-			'event_id'      => get_post_meta( $attendee->ID, self::ATTENDEE_EVENT_KEY, true ),
-			'ticket_name'   => ! empty( $product ) ? $product->post_title : false,
-			'holder_name'   => get_post_meta( $attendee->ID, $this->full_name, true ),
-			'holder_email'  => get_post_meta( $attendee->ID, $this->email, true ),
-			'order_id'      => $attendee->ID,
-			'ticket_id'     => $ticket_unique_id,
-			'qr_ticket_id'  => $attendee->ID,
-			'security_code' => $security,
-
-			// Attendee Meta
-			'attendee_meta' => $meta,
-		) );
+		$attendee_data = array_merge(
+			$this->get_order_data( $attendee->ID ),
+			[
+				'optout'             => $optout,
+				'ticket'             => $product_title,
+				'attendee_id'        => $attendee->ID,
+				'security'           => $security,
+				'product_id'         => $product_id,
+				'check_in'           => $checkin,
+				'order_status'       => $status,
+				'order_status_label' => $status_label,
+				'user_id'            => $user_id,
+				'ticket_sent'        => $ticket_sent,
+
+			// Fields for Email Tickets.
+				'event_id'           => get_post_meta( $attendee->ID, self::ATTENDEE_EVENT_KEY, true ),
```

Comment:
:no_entry_sign: Multi-line array item not aligned correctly; expected 20 spaces, but found 16

<sub>hash: e0192c5aef3a0b546f84616534fa6c35</sub>